packages.bzl

```diff
@@ -17,6 +17,7 @@ _CDK_ENTRY_POINTS = [
     "layout",
     "bidi",
     "portal",
+    "private",
```

Comment:
What's private for?

Reply:
looks like CDK has a new private package: https://github.com/angular/components/tree/main/src/cdk/private